collections/src/main/scala/strawman/collection/SortedMap.scala

```diff
@@ -10,7 +10,9 @@ import scala.{Boolean, Int, Option, Ordering, PartialFunction, Serializable, `in
 /** Base type of sorted sets */
 trait SortedMap[K, +V]
   extends Map[K, V]
-    with SortedMapOps[K, V, SortedMap, SortedMap[K, V]]
+    with SortedMapOps[K, V, SortedMap, SortedMap[K, V]] {
+  def unsorted: Map[K, V]
```

Comment:
Why don’t you implement the operation directly here?

Reply:
I got it. Thank you for pointing out! :) I will push another commit.